docs/databases/postgresql/ubuntu-12-04-precise-pangolin.md

```diff
@@ -2,22 +2,23 @@
 author:
   name: Linode
   email: docs@linode.com
-description: 'Using the PostgreSQL relational database server with Ubuntu 12.04 LTS (Precise Pangolin).'
-keywords: 'postgresql,postgresql ubuntu 12.04,postgreql lucid,postgresql database,open source database,relational database'
+description: 'Use PostgreSQL relational DBMS on Ubuntu 12.04.'
+keywords: 'postgresql,ubuntu 12.04,postgresql database,postgresql ubuntu, open source database,relational database,DBMS,create database,create table'
+alias: ['databases/postgresql/ubuntu-12-04-precise-pangolin']
 license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
 modified: Monday, October 8th, 2012
 modified_by:
   name: Linode
 published: 'Monday, October 8th, 2012'
-title: 'Use PostgreSQL Relational Databases on Ubuntu 12.04 LTS (Precise Pangolin)'
+title: 'Use PostgreSQL Relational DBMS on Ubuntu 12.04'
 external_resources:
  - '[PostgreSQL Online Documentation](http://www.postgresql.org/docs/)'
  - '[psql manual page](http://www.rootr.net/man/man/psql/1)'
 ---
 
-The [PostgreSQL](http://www.postgresql.org/) relational database system is a fast, scalable, and standards-compliant open source database platform. This guide will help you install and configure PostgreSQL on your Ubuntu 12.04 LTS (Precise Pangolin) VPS. We assume you've followed the steps detailed in our [getting started guide](/docs/getting-started/), and that you're logged into your Linode VPS as root via SSH.
+The [PostgreSQL](http://www.postgresql.org/) relational database system (DBMS) is a fast, scalable, and standards-compliant open source database platform. This guide will help you install and configure PostgreSQL on your Ubuntu 12.04 LTS (Precise Pangolin) VPS. We assume you've followed the steps detailed in our [getting started guide](/docs/getting-started/), and that you're logged into your Linode VPS as root via SSH.
```

Comment:
DBMS stands for Database Management Software. Not applicable here, since this is a DB itself, not management software.